File: src/FE/jts/src/components/statistics/judoka-view.js
import { React } from 'react';

const JudokaView = ({ index, judokaId, when }) => {
    return (
        <tr>
            <td>{index}</td>
            <td>{judokaId}</td>
            <td>{when}</td>
        </tr>
    )
};

export default JudokaView;

File: src/BE/Domain/Statistics/Models/JudokaView.cs
namespace JTSystem.Domain.Statistics.Models
{
    using JTSystem.Domain.Common.Models;
    using System;

    public class JudokaView : Entity<int>
    {
        internal JudokaView(int judokaId)
        {
            this.JudokaId = judokaId;
            this.When = DateTime.UtcNow;
        }

        public int JudokaId { get; }

        public DateTime When { get; }
    }
}
